hack/test-cmd.sh

```diff
@@ -246,112 +246,16 @@ os::test::junit::declare_suite_start "cmd/config"
-os::cmd::expect_success 'oc logout'
-# logs in skipping certificate check
-os::cmd::expect_success "oc login ${KUBERNETES_MASTER} --insecure-skip-tls-verify -u test-user -p anything"
-# logs in by an existing and valid token
-temp_token=$(oc config view -o template --template='{{range .users}}{{ index .user.token }}{{end}}')
-os::cmd::expect_success_and_text "oc login --token=${temp_token}" 'using the token provided'
-os::cmd::expect_success 'oc logout'
-# properly parse server port
-os::cmd::expect_failure_and_text 'oc login https://server1:844333' 'Not a valid port'
-# properly handle trailing slash
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
-# create a new project
-os::cmd::expect_success "oc new-project project-foo --display-name='my project' --description='boring project description'"
-os::cmd::expect_success_and_text "oc project" 'Using project "project-foo"'
-# new user should get default context
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u new-and-unknown-user -p anything"
-os::cmd::expect_success_and_text 'oc config view' "current-context.+/${API_HOST}:${API_PORT}/new-and-unknown-user"
-# denies access after logging out
-os::cmd::expect_success 'oc logout'
-os::cmd::expect_failure_and_text 'oc get pods' '"system:anonymous" cannot list pods'
-
-# log in as an image-pruner and test that oadm prune images works against the atomic binary
-os::cmd::expect_success "oadm policy add-cluster-role-to-user system:image-pruner pruner --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'"
-os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u pruner -p anything"
-# this shouldn't fail but instead output "Dry run enabled - no modifications will be made. Add --confirm to remove images"
-os::cmd::expect_success 'oadm prune images'
-
-# log in and set project to use from now on
-VERBOSE=true os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
-VERBOSE=true os::cmd::expect_success 'oc get projects'
-VERBOSE=true os::cmd::expect_success 'oc project project-foo'
-os::cmd::expect_success_and_text 'oc config view' "current-context.+project-foo/${API_HOST}:${API_PORT}/test-user"
-os::cmd::expect_success_and_text 'oc whoami' 'test-user'
-os::cmd::expect_success_and_text "oc whoami --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'" 'system:admin'
-os::cmd::expect_success_and_text 'oc whoami -t' '.'
-os::cmd::expect_success_and_text 'oc whoami -c' '.'
-
-# test config files from the --config flag
-os::cmd::expect_success "oc get services --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'"
-
-# test config files from env vars
-os::cmd::expect_success "KUBECONFIG='${MASTER_CONFIG_DIR}/admin.kubeconfig' oc get services"
```

Comment:
none of the following tests would ever work with this functionality broken so I'm not really sure this is a useful test